src/SDKs/IotCentral/Management.IotCentral/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd iotcentral/resource-manager %*
```

Comment:
generate.cmds are not needed anymore, all code generation must go through generate.ps1s

Reply:
removed